project/README.md

```diff
@@ -14,6 +14,16 @@
 
 
 ## React
+[Jayesh Tambe](https://github.com/jtambe)
+
+**Major**
+
+* [Developed example code for DOM manipulation using react](https://github.com/facebook/react/pull/9683)
+
+**Minor**
+
+* [I had added documentation for setting up react environment on ubuntu machine](That link is no longer in the repo)
```

Comment:
It was part of wiki. I did not do documentation as a pull request. Let me check their wiki history..